src/components/course/enrollment/tests/utils.test.js

```diff
@@ -125,19 +125,19 @@ describe('canUserRequestSubsidyForCourse', () => {
     },
     {
       subsidyRequestConfiguration: disabledSubsidyRequests,
-      subsidyRequestCatalogsApplicableToCourse: new Set(['catalog-uuid']),
+      subsidyRequestCatalogsApplicableToCourse: Array.from(new Set(['catalog-uuid'])),
```

Comment:
nit: could opt to just use `['catalog-uuid']` instead of defining `Set` only to immediately convert it to an array.